### ISSUE TITLE
a trx with no output is present on mc4, breaks the explorer

### ISSUE THREAD

**Issue by ashisherc** [input-output-hk/cardano-explorer-app#329]
Summary
A particular transaction has no outputs on the chain, I believe the transaction should have a valid utxo.
transaction hash: 05e5bb869d0a71e3689d7a7d2bcde70494e370df0e8958fb338562d94a7dddb6

Steps to reproduce

Go to https://explorer.mainnet-candidate-4.dev.cardano.org/en/
open chrome console
search 05e5bb869d0a71e3689d7a7d2bcde70494e370df0e8958fb338562d94a7dddb6
See error
Expected behavior
A transaction should have a valid utxo with inputs and outputs

**Comment by dcoutts** [comment]
Note that it is legitimate for a transaction to have no outputs. The explorer is supposed to handle such txs correctly.

**Comment by DominikGuzei** [comment]
Thanks for the report, we could reproduce the issue and we are working on a fix 👍

**Comment by rhyslbw** [comment]
@DominikGuzei 
https://github.com/input-output-hk/cardano-graphql/pull/259#discussion_r461591063
```graphql
query orderedTransactionsInBlock(
          $hash: Hash32HexString!
      ) {
          transactions(
              where: { hash: { _eq: $hash } },
          ) {
              outputs {
                  address
                  value
              }
              outputs_aggregate {
                  aggregate {
                      count
                  }
              }
              inputs_aggregate {
                  aggregate {
                      count
                  }
              }
          }
      }
```
``` typescript
expect(result.data.transactions.length).toBe(1)
expect(result.data.transactions[0].outputs_aggregate.aggregate.count).toBe('0')
expect(result.data.transactions[0].outputs).toEqual([])
expect(result.data.transactions[0].inputs_aggregate.aggregate.count).toBe('1')
```

**Comment by rhyslbw** [comment]
UI issue is resolved in #331. We have a minor defect limited to a console error, so will leave open until complete resolution
